scripts/deploy_oak_functions_loader

```diff
@@ -22,7 +22,7 @@ docker push "${FUNCTIONS_DOCKER_IMAGE_NAME}:latest"
 
 # Build and push the `weather_lookup` example application built on Oak Functions.
 
-cargo -Zunstable-options build --release \
+./scripts/docker_run cargo -Zunstable-options build --release \
   --target=wasm32-unknown-unknown \
   --manifest-path=./oak_functions/examples/weather_lookup/module/Cargo.toml \
   --out-dir=./oak_functions/examples/weather_lookup/bin
```

Comment:
Isn't there a runner command for this too?

Reply:
I don't think we have `runner` commands for building `.wasm` files. It is being done as part of the `run-functions-examples`. Should I just call the `run-functions-examples`, and even skip creating the docker image for the loader?